### PR TITLE
Add UserController

### PR DESCRIPTION
**Pull request by ifteakhor**
- Add new Controller for user
- Implement getById method
- Implement Edit method

### CODE REVIEW

File: server/Alumni/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        if (user is null)
+        {
+			return NotFound();
+    	}
+
+        return user;
+    }
+}
+
+public class EditPersonalInfoDTO
```
Comment (veyxov) [inline server/Alumni/Controllers/UserController.cs]: Extract this to Dtos/ direcctory

File: server/Alumni/Controllers/UserController.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+		_context.SaveChanges();
+
+        return user;
+    }
+
+    [HttpGet("{id}")]
+    public ActionResult<User> Get(int id)
+    {
+        var user = _context.Users.FirstOrDefault(
```
Comment (veyxov) [inline server/Alumni/Controllers/UserController.cs]: please use async for DB operations

File: server/Alumni/Controllers/UserController.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            u => u.Id == id
+        );
+
+		if (user is null) {
+			return NotFound();
+		}
+
+        user.Name = dto.Name;
+		user.Login = dto.Email;
```
Comment (veyxov) [inline server/Alumni/Controllers/UserController.cs]: You can format code using [csharpier](https://csharpier.com/)

File: server/Alumni/Controllers/UserController.cs [hunk 4]
```diff
@@ -0,0 +1,58 @@
+using Microsoft.AspNetCore.Mvc;
+using Alumni.Models;
+
+
+namespace Alumni.Controllers;
+
+[ApiController]
+[Route("api/users")]
```
Comment (veyxov) [inline server/Alumni/Controllers/UserController.cs]: I want to use 
- GET - `api/users` get all users (REST)
- POST - `api/user` create new user
when add a 'controller level route' it decreases naming flexibility

**Review by veyxov — CHANGES_REQUESTED** [review]
Please take a look at the comments, feel free to ask questions

File: server/Alumni/Controllers/UserController.cs (hunk 1 above)
Comment (ifteakhor) [reply server/Alumni/Controllers/UserController.cs]: fixed

File: server/Alumni/Controllers/UserController.cs (hunk 2 above)
Comment (ifteakhor) [reply server/Alumni/Controllers/UserController.cs]: done

File: server/Alumni/Controllers/UserController.cs (hunk 4 above)
Comment (ifteakhor) [reply server/Alumni/Controllers/UserController.cs]: done

File: server/Alumni/Controllers/UserController.cs (hunk 3 above)
Comment (ifteakhor) [reply server/Alumni/Controllers/UserController.cs]: done

**Pull request merged**